versions/3.0.md

```diff
@@ -761,67 +751,83 @@ schema:
 <a name="requestBodyExamples"></a>examples | [Examples Object](#examplesObject) | Examples of the request body, referenced by media type.
 <a name="requestBodyRequired"></a>required | `boolean` | Determines if the request body is required in the request. Defaults to `true`.
 
 
 ##### Patterned Fields
 Field Pattern | Type | Description
 ---|:---:|---
+<a name="requestBodyRepresentation"></a>`*` | [Schema Object](#schemaObject) | The schema defining the request body.
 <a name="parameterExtensions"></a>^x- | Any | Allows extensions to the OpenAPI Schema. The field name MUST begin with `x-`, for example, `x-internal-id`. The value can be `null`, a primitive, an array or an object. See [Vendor Extensions](#vendorExtensions) for further details.
 
 
 ##### Request Body Examples
 
 A request body with a referenced model definition.
-```js
+```json
 {
   "description": "user to add to the system",
-  "schema": {
-    "$ref": "#/definitions/User"
-  },
-  "examples": {
-        "application/json": {
-          "$ref": 'http://foo.bar#/examples/address-example.json'
-        'application/xml':
-          $ref: 'http://foo.bar#/examples/address-example.xml'
-        'text/plain':
-          $ref: 'http://foo.bar#/examples/address-example.txt'
-        default:
-          $ref: 'http://foo.bar#/examples/address-example.whatever'
+  "representations": {
+    "application/json": {
+      "schema": {
+        "$ref": "#/definitions/User"
+      },
+      "example": {
+        "$ref": "http://foo.bar#/examples/address-example.json"
+      }
+    },
+    "application/xml": {
+      "schema": {
+        "$ref": "#/definitions/User"
+      },
+      "example": {
+        "$ref": "http://foo.bar#/examples/address-example.xml"
+      }
+    },
+    "*": {
+      "example": {
+        $ref: "http://foo.bar#/examples/address-example.whatever"
+      }
+    }
+  }
 }
 ```
 
 ```yaml
 description: user to add to the system
-schema:
-  $ref: '#/definitions/User'
-examples:
-        'application/json':
-          $ref: 'http://foo.bar/examples/user-example.json'
-        'application/xml':
-          $ref: 'http://foo.bar/examples/user-example.xml'
-        'text/plain':
-          $ref: 'http://foo.bar/examples/user-example.txt'
-        default:
-          $ref: 'http://foo.bar/examples/user-example.whatever'
+representations:
+  'application/json':
+    schema:
+      $ref: '#/definitions/User'
+    example:
+      $ref: 'http://foo.bar/examples/user-example.json'
+  'application/xml':
+    schema:
+      $ref: '#/definitions/User'
+    example:
+      $ref: 'http://foo.bar/examples/user-example.xml'
+  '*':
+    schema:
+      $ref: 'http://foo.bar/examples/user-example.whatever'
 ```
 
 A body parameter that is an array of string values:
-```js
+```json
 {
   "description": "user to add to the system",
-  "schema": {
-    "type": "array",
-    "items": {
-      "type": "string"
+  "representations": {
```

Comment:
The use of a JSON array schema with `text/plain` is confusing. If it is an array of strings, it should be application/json. if it is a text (i.e. one user per line) then it is not an array. (Repeated in the YAML example.)